scipy/stats/_mstats_extras.py

```diff
@@ -15,7 +15,7 @@
 
 
 import numpy as np
-from numpy import float_, int_, ndarray
+from numpy import double, int_, ndarray
```

Comment:
Why is it changing to `double` in this file?

Reply:
Because `float_` is being removed as it's an [other alias](https://numpy.org/devdocs/reference/arrays.scalars.html#other-aliases) - `double` is a dtype that `float_` aliases. I can also change it sized dtype, `float64`.

Reply:
+1 for changing to `float64` here.

Reply:
Sure! I changed it to `float64`.